langchain/client/runner_utils.py

```diff
@@ -325,7 +325,7 @@ async def _callbacks_initializer(
     """
     callbacks: List[BaseTracer] = []
     if project_name:
-        callbacks.append(LangChainTracer(project_name=project_name))
+        callbacks.append(LangChainTracer(project_name=project_name, client=client))
```

Comment:
This shares the same client across all threads, so multiple threadpool executors all sharing a single client.  I'm assuming I should create a new client for each tracer here that uses the same parameters @nfcampos

Reply:
Updated to share an executor.